charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -366,10 +366,14 @@ data:
       - source_labels: [ __meta_kubernetes_pod_name ]
         action: keep
         regex: vpn-shoot-(.+)
-      - source_labels: [ __meta_kubernetes_pod_name ]
+      - source_labels: [ __meta_kubernetes_pod_container_name ]
+        regex: (vpn-seed-.*|vpn-seed)
```

Comment:
`vpn-shoot` pod has containers named `vpn-shoot` or `vpn-shoot-0`, `vpn-shoot-1`...

Why do use `action: replace` and `replacement: $1`? Would it not work analogous to the pod with `action: keep` and no `replacement`?

Reply:
In HA mode, `vpn-shoot` pod has two containers `vpn-shoot-s0` and `vpn-shoot-s1`
The `action: keep` will keep both containers name which match the regular expression. The configuration i used will select the first container name "vpn-seed-s0" as prometheus target. 
I am assuming we will probe once for one `vpn-shoot` pod. If we get the two containers of one vpn-shoot pod, it will probe twice. I am not sure if this is needed? 
@ScheererJ How do you think?